src/CNN/tools/Preprocessing.py

```diff
@@ -133,7 +133,7 @@ def calcProtVoxel(self, x, y, z, protPath, number, altProtPath):
                     )
                 except:
                     f = open("../../Data/prep_log.txt", "a")
-                    f.writelines('Protein ' + protPath + ' leads to errors! Proteinnumber: ' + str(number) + '\n')
```

Comment:
The number is very usefull, because you know, which starting point you should use after the fix. This avoids recalc everything.